### ISSUE TITLE
[bridges][AHs] Include ExportMessage fees in XCM message from AH to BH

### ISSUE THREAD

**Issue by acatangiu** [polkadot-fellows/runtimes#392]
XCM execution on Bridge Hub for messages coming from Asset Hub is being paid from AHs sovereign account, requiring periodic top-up of said sovereign account.

We can improve that by including the execution fees in the message (teleport DOT alongside ExportMessage).

**Comment by vgeddes** [comment]
I think we will do something like this for the Ethereum->BridgeHub flow, mainly so that we don't have to manually rebalance the collected fees.

Specifically:
1. On Ethereum, user pays ETH along with their message
2. ETH is locked into SA for AH
3. Fee is included in message
4. When delivered to BridgeHub, an XCM is sent to AH to mint the WETH to an SA controlled by the bridge
5. A mechanism will be added to BridgeHub to allow relayers to claim their WETH reward on AssetHub

Then relayers can use AssetHub DEX to convert the WETH back into DOT if they want.

**Comment by acatangiu** [comment]
Yeah, that sounds good. And this works even for new bridge lanes dedicated to other parachains. Relayer rewards are always minted on AH and claimed through BH (by providing some beneficiary AH address to be credited or similar).

**Comment by yrong** [comment]
> Relayer rewards are always minted on AH and claimed through BH

Or as mentioned in https://github.com/Snowfork/polkadot-sdk/pull/160#discussion_r1683682152 a LockAssets|UnlockAssets make more sense here?
